keyboards/aidansmithdotdev/fine40/fine40.c

```diff
@@ -67,19 +67,4 @@ bool oled_task_kb(void) {
 	//render_logo();
 	return(true);
 }
```

Comment:
This should remain to provide default behaviour on configurator
 ```suggestion
}
#endif

#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) {
        return false;
    }
	// Volume control
	if (clockwise) {
		tap_code(KC_VOLU);
	} else {
		tap_code(KC_VOLD);
	}
	return false;
}
```